### PR TITLE
create models according to api specfication

### PR DESCRIPTION
**Pull request by rekagiday**

### CODE REVIEW

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/model/building/Barrack.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package com.greenfox.tribesoflagopusandroid.model.building;
+
+/**
+ * Created by User on 2017. 06. 14..
+ */
+
+public class Barrack extends Building {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/model/building/Barrack.java]: @rekagiday we dont need this

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/model/building/Farm.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package com.greenfox.tribesoflagopusandroid.model.building;
+
+/**
+ * Created by User on 2017. 06. 14..
+ */
+
+public class Farm extends Building {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/model/building/Farm.java]: @rekagiday we dont need this

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/model/building/Mine.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
+package com.greenfox.tribesoflagopusandroid.model.building;
+
+/**
+ * Created by User on 2017. 06. 14..
+ */
+
+public class Mine extends Building {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/model/building/Mine.java]: @rekagiday we dont need this

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/model/building/Townhall.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
+package com.greenfox.tribesoflagopusandroid.model.building;
+
+/**
+ * Created by User on 2017. 06. 14..
+ */
+
+public class Townhall extends Building {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/model/building/Townhall.java]: @rekagiday we dont need this

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/model/resource/Food.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
+package com.greenfox.tribesoflagopusandroid.model.resource;
+
+/**
+ * Created by User on 2017. 06. 14..
+ */
+
+public class Food extends Resource {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/model/resource/Food.java]: @rekagiday we dont need this

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/model/resource/Gold.java [hunk 6]
```diff
@@ -0,0 +1,46 @@
+package com.greenfox.tribesoflagopusandroid.model.resource;
+
+/**
+ * Created by User on 2017. 06. 14..
+ */
+
+public class Gold extends Resource {
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/model/resource/Gold.java]: @rekagiday we dont need this

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Comment by rekagiday** [comment]
removed marked files

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**